Core/clim-core/dialog.lisp

```diff
@@ -245,8 +245,15 @@ accept of this query")))
                                       :record-type 'accepting-values-record)
                       (if align-prompts
                           (formatting-table (stream)
-                            (setf return-value (funcall body *accepting-values-stream*)))
-                            (setf return-value (funcall body *accepting-values-stream*)))
+                            #1=(setf return-values
+                                     (multiple-value-list
+                                      (funcall body *accepting-values-stream*))))
+                          #1#)
```

Comment:
why are we using this reader macro stuff here?

Reply:
I've taken this habit from Peter Norvig Lisp style guide (and I generally agree with that - if you have exactly same form repeated in a slightly different context one after another repeating text may lead to bugs, because someone may fix the form in one place but not in the another). If you don't agree, I can change that (so the same form is copied verbatim instead of using `#1#`).